pkgs/development/libraries/phonon/backends/gstreamer.nix

```diff
@@ -67,7 +58,13 @@ stdenv.mkDerivation rec {
     qttools
   ];
 
-  cmakeFlags = [
-    "-DCMAKE_BUILD_TYPE=${if debug then "Debug" else "Release"}"
-  ];
+  cmakeBuildType = if debug then "Debug" else "Release";
+
+  meta = with lib; {
+    homepage = "https://phonon.kde.org/";
+    description = "GStreamer backend for Phonon";
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ ttuegel ];
+    license = licenses.lgpl21;
+  };
```

Comment:
Ah, I see you _moved_ meta. IMHO it's a good idea to keep commits strictly on topic and not do unrelated refactoring.